front-end-side/src/Pages/Data.js

```diff
@@ -126,19 +127,18 @@ const EnterData = () => {
                     <tbody>
                         {
                             statusForFileData ? (
-
-                                fileData.data.slice(0, size).map((item, index) => {
-                                    const { date, seller, purpose, amount } = item;
+                                fileData.data.sort((a, b) => a.date > b.date ? 1 : -1).slice(0, size).map((item, index) => {
```

Comment:
Do we need index here?

Reply:
No, since we now find records by id and not index, I will delete it